components/_components.scss

```diff
@@ -0,0 +1,242 @@
+// .fa-twitter {
+//   font-size: 32px;
+// }
+.tweet-btn {
+  width: 100%;
+  display: flex;
+  //   justify-content: center;
+  #tweet-btn {
+    text-align: center;
+    border: transparent;
+    border-radius: 200px;
+    color: #fff;
+    font-size: 2rem;
+    padding: 1.4rem 8rem;
+    background-color: $color-main;
+    &:hover {
+      background-color: $color-main-dark;
+      transition: all 0.3s;
+      cursor: pointer;
+    }
+    @include respond(tab-land) {
```

Comment:
What does @include do?

Reply:
its a sass feature which is call a mixin function which has some pre-defined properties